securedrop/source_templates/index.html

```diff
@@ -26,7 +26,7 @@
            See _source_index.sass for a more full understanding. #}
         <div class="index-wrap">
           <div class="header">
-            <img src="{{ url_for('main.select_logo') }}" alt="SecureDrop" class="logo-image">
+            <img src="{{ url_for('main.select_logo') }}" alt="Logo Image" class="logo-image">
```

Comment:
Consider using  `gettext()` to allow this string to be translated. (See line 32 below for an example).

Reply:
Made `Logo Image` string available for translation via https://github.com/freedomofpress/securedrop/pull/4980/commits/a9f8b9abdab6845f6bde50575272c12e5849d2b6.